xbmc/InfoScanner.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *      Copyright (C) 2015 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with KODI; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+#include <string>
+#include <vector>
+
+class InfoScanner {
```

Comment:
I missed this one:( To avoid more ping-pongs with the style nits I am trying to configure a formatter following http://kodi.wiki/view/Official:Code_guidelines_and_formatting_conventions

Any other conventions the wiki may be missing?